cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -217,12 +217,18 @@ <h2 class="h4">
                                                 {% endif %}
 
                                                 {% if counselor.phone1 %}
-                                                <b>Phone:</b> {{ counselor.phone1 }}
+                                                <b>Phone:</b>
+                                                <a class="a-link" href="tel:+1-{{ counselor.phone1 }}">
+                                                    <span class="a-link_text">{{ counselor.phone1 }}</span>
```

Comment:
I don't think we need the `a-link` class or the whole `span.a-link_text` element, since these links don't have icons.

Reply:
Good catch. I was following the formatting of the website URL, but that does have an icon. Updated.